Functions.Templates/Templates-v2/EventHubTrigger-Python/blueprint.py

```diff
@@ -0,0 +1,3 @@
+from azure.functions import Blueprint
+
+$(FILE_NAME_INPUT) = Blueprint()
```

Comment:
There is no `$(FILE_NAME_INPUT)` token defined in template.json - did you mean `$(BLUEPRINT_FILENAME)?`

Reply:
Updated to BLUEPRINT_FILENAME